docs/content/docs/dev/table/sql/queries/time-travel.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Time Travel
+weight: 18
+type: docs
+---
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+# Time Travel
+
+{{< label Batch >}} {{< label Streaming >}}
+
+`TIME TRAVEL` is an SQL syntax used for querying historical data. It allows users to specify a point in time, query the corresponding table data.
+
+<span class="label label-danger">Attention</span> Currently, `TIME TRAVEL` requires corresponding catalog that the table belongs to implement the {{< gh_link file="flink-table/flink-table-common/src/main/java/org/apache/flink/table/catalog/Catalog.java" name="getTable(ObjectPath tablePath, long timestamp)" >}} method。
+
+```sql
+SELECT select_list FROM paimon_tb FOR SYSTEM_TIME AS OF TIMESTAMP '2023-07-31 00:00:00'
+```
+
+## Expression Description
+
+<span class="label label-danger">Attention</span> `TIME TRAVEL` currently only supports some constant expressions and does not support the use of functions or udf.
```

Comment:
```suggestion
<span class="label label-danger">Attention</span> `Time travel` currently only supports some constant expressions and does not support the use of functions or udf.
```

Reply:
Also, I don't think it does not support all the use of functions or udf. From the test of `ExpressionReductionRulesTest`, I believe it can reduce some simple expression even with udf.